styles.css

```
@@ -1,3 +1,564 @@
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 85%);
+    z-index: 2;
+}
+
+.nav-bar .nav-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    top: 0;
+    left: 0;
+    padding: 20px 80px 20px 20px;
+}
+
+.nav-list a {
+    padding: 1rem;
+    color: var(--magnolia);
+}
+
+.intro {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    padding: 50px;
+    width: 100%;
+}
+
+.intro > p {
+    color: var(--magnolia);
+    line-height: 2.5rem;
+    margin-top: 10px;
+}
+
+button {
+    display: inline-block;
+    border-radius: 25px;
+    background: var(--french-rose);
+    color: var(--magnolia);
+    box-shadow: rgba(25, 25, 25, 0.04) 0 0 10px 0, rgba(0, 0, 0, 0.1) 0 5px 5px 0;
+    text-transform: uppercase;
+    border: none;
+    padding: 1rem 2rem;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    cursor: pointer;
+}
+
+.squiggle-line-white {
+    background-image: url("./assets/squiggle_line_stripe.svg");
+    height: 90px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+}
+
+/*///////////// 4.2 Introduction  /////////////*/
+
+.flex-outer-container-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+}
+
+.company-intro {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+    padding: 30px;
+    margin-top: 30px;
+}
+
+.questions {
+    color: var(--night);
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    padding-top: 50px;
+    padding-bottom: 60px;
+    gap: 5px;
+}
+
+.company-intro > p > span {
+    color: var(--chrysler-blue);
+}
+
+.company-intro > h4 {
+    padding-top: 15px;
+}
+
+/*///////////// 4.3 Project Section /////////////*/
+
+.projects {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: var(--light-gray);
+    padding-bottom: 150px;
+    padding-top: 150px;
+}
+
+.projects > article {
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 25px 5px 25px;
+    text-align: left;
+    width: 90%;
+    margin: 2rem auto 0 auto;
+}
+
+.projects > article > div {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-grow: 0;
+    flex-shrink: 1;
+    justify-content: center;
+    padding: 20px;
+}
+
+.projects > article > span > img {
+    width: 100%;
+    box-shadow: rgba(50, 50, 93, 0.25) 0 50px 100px -20px, rgba(0, 0, 0, 0.3) 0 30px 60px -30px;
+}
+
+.project-image {
+    flex-basis: 1500px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+.projects > article > span {
+    padding: 50px -10px 50px -10px;
```

Dit zou je nog korter kunnen schrijven
```suggestion
    padding: 50px -10px;
```